trajectory.py

```diff
@@ -211,13 +230,18 @@ def _compute_speed(self, row):
         return dist_meters / row['delta_t'].total_seconds()
 
     def add_direction(self, overwrite=False):
+        """Add direction column and values to Trajectory object's DataFrame."""
         if DIRECTION_COL_NAME in self.df.keys() and not overwrite:
             raise RuntimeError('Trajectory already has direction values! Use overwrite=True to overwrite exiting values.')
         self.df['prev_pt'] = self.df.geometry.shift()
         self.df[DIRECTION_COL_NAME] = self.df.apply(self._compute_heading, axis=1)
         self.df.at[self.get_start_time(), DIRECTION_COL_NAME] = self.df.iloc[1][DIRECTION_COL_NAME]
 
     def add_speed(self, overwrite=False):
+        """Add speed column and values to Trajectory object's DataFrame.
+
+        Calculation is in measurement unit of CRS divided by total seconds.
```

Comment:
This description is not completely correct since get_length() calls _compute_speed() which returns speed in CRS units per second, except for trajectories where .is_latlon() is true in which case it returns meters per second.

Reply:
Made suggested change here